rfcs/text/0032-definition-of-ecs-compliance.md

```diff
@@ -46,14 +45,11 @@ ECS-compliant events SHOULD:
 
 ECS-compliant events MAY:
 
+* store the entire raw, original event in `event.original`. Disable indexing and doc_values on `event.original` to reduce store.
 * use custom fields alongside ECS fields in an event. Use proper names for custom fields over generic concept names. Proper names reduce the chance of a future conflict. Nest custom fields inside an object and not leaf fields at the base of the event.
```

Comment:
This is vague to me. I don't understand the guidance. What does it mean by "proper names"? Does this mean a proper noun as in a capitalized key name? If so consider directly saying to use capitalized key names to avoid future conflicts because ECS only uses lowercase names.

Reply:
Capitalized field names, but also using a project or company namespace for custom fields(`nginx`, `aws`, etc.).

Yes, "proper name" == "proper noun"; I'll rephrase.